runtime/vm/ContinuationHelpers.cpp

```diff
@@ -204,7 +204,11 @@ yieldContinuation(J9VMThread *currentThread)
 {
 	BOOLEAN result = TRUE;
 	J9VMContinuation *continuation = currentThread->currentContinuation;
+	j9object_t continuationObject = J9VMJAVALANGTHREAD_CONT(currentThread, currentThread->carrierThreadObject);
+	ContinuationState volatile *continuationStatePtr = VM_VMHelpers::getContinuationStateAddress(currentThread, continuationObject);
 	Assert_VM_notNull(currentThread->currentContinuation);
+	Assert_VM_false(VM_VMHelpers::isPendingToBeMounted(*continuationStatePtr));
+	Assert_VM_notNull(VM_VMHelpers::getCarrierThread(*continuationStatePtr));
```

Comment:
we already have a stronger assertion later in line 238 - do we really need this one? although we could move that one here